sandbox/package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why do you have `package-lock.json` ni your sandbox folder? Maybe you have created it by mistake?